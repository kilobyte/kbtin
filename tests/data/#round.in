#ali m {#math x $0;#show $0 â†’ $x}
m #round 0
m #round 1
m #round 1.0
m #round 1.1
m #round 0.1
m #round 0.9
m #round -1.0
m #round -0.1
m #round -0.9
m #round -0.1
m #round 0.4999999
m #round 0.5
m #round 0.5000001
m #round 1.4999999
m #round 1.5
m #round 1.5000001
m #round -0.4999999
m #round -0.5
m #round -0.5000001
m #round -1.4999999
m #round -1.5
m #round -1.5000001
m #round 2147483647
m #round -2147483648
