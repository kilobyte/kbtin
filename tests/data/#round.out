#round 0 → 0
#round 1 → 1
#round 1.0 → 1
#round 1.1 → 1
#round 0.1 → 0
#round 0.9 → 1
#round -1.0 → -1
#round -0.1 → 0
#round -0.9 → -1
#round -0.1 → 0
#round 0.4999999 → 0
#round 0.5 → 1
#round 0.5000001 → 1
#round 1.4999999 → 1
#round 1.5 → 2
#round 1.5000001 → 2
#round -0.4999999 → 0
#round -0.5 → -1
#round -0.5000001 → -1
#round -1.4999999 → -1
#round -1.5 → -2
#round -1.5000001 → -2
#round 2147483647 → 2147483647
#round -2147483648 → -2147483648
